Applications/Source/Palette Designer/Program.cs

```diff
@@ -1,25 +1,41 @@
 ﻿#region BSD License
 /*
- * 
+ *
  * Original BSD 3-Clause License (https://github.com/ComponentFactory/Krypton/blob/master/LICENSE)
  *  © Component Factory Pty Ltd, 2006 - 2016, (Version 4.5.0.0) All rights reserved.
- * 
+ *
  *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
- *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2017 - 2023. All rights reserved. 
- *  
+ *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2017 - 2023. All rights reserved.
+ *
  */
 #endregion
 
+using System.Runtime.InteropServices;
+
 namespace PaletteDesigner
 {
     internal static class Program
     {
+#if NET48 || NET481 || NET472 || NET471 || NET47 || NET462 || NET461 || NET46
```

Comment:
There is an all encompassing netFramewrok define that is easier to read (And include 4.8.1)